README.md

```diff
@@ -1,15 +1,61 @@
 # sme_finance_application_schema
+
+https://travis-ci.org/FundingOptions/sme-finance-application-schema.svg?branch=master
+
 ## Dislaimer
-These schema are released to facilitate industry implementation of the Government referral scheme. We actively welcome queries, feedback and suggestions from any interested parties, which can be provided via our GitHub repository or to info@fundingoptions.com.
+These schema are released to:
+* Allow Funding Options to communicate with Partners (including Lenders)
+* Facilitate industry implementation of the Government referral scheme
+We actively welcome queries, feedback and suggestions from any interested parties, which can be provided via [our GitHub repository](https://github.com/FundingOptions/sme-finance-application-schema) or to info@fundingoptions.com.
```

Comment:
can this point at one of our groups that **actually** goes to some of the developers? (likewise for any public emails we have on github)